src/isIsogram.test.js

```diff
@@ -6,4 +6,47 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should return true if users nothing enters`, () => {
+    expect(isIsogram(''))
+      .toBeTruthy();
+  });
+
+  it(`should return true if users enters isogram`, () => {
+    expect(isIsogram('beach'))
+      .toBeTruthy();
+  });
+
+  it(`should return false if users enters repeating letters`, () => {
+    expect(isIsogram('toothpick'))
+      .toBeFalsy();
+  });
+
+  it(`should return false if users enters word with 
+    identical letters in different cases`, () => {
+    expect(isIsogram('toOthpick'))
+      .toBeFalsy();
+  });
+
+  it(`should return false if users enters word repeating 
+    consecutive parts`, () => {
+    expect(isIsogram('bilabila'))
+      .toBeFalsy();
+  });
+
+  it(`should return true if users enters one letter`, () => {
+    expect(isIsogram('q'))
+      .toBeTruthy();
+  });
+
+  it(`should return true if users enters long isogram`, () => {
+    expect(isIsogram('abcdefghijklmnopqrstuvwxyz'))
+      .toBeTruthy();
+  });
+
+  it(`should return true if users enters two identical letters 
```

Comment:
Rewrite the test description to 'should return true if two identical letters from different alphabets are entered' and remove the line break for better readability 📖

Reply:
I can't remove the line break because the line's length is more than 80 characters and an error message is displayed "Line 47 exceeds the maximum line length of 80."